pandas/tseries/tests/test_base.py

```diff
@@ -32,7 +32,7 @@ def test_ops_properties(self):
                                    'is_month_start', 'is_month_end',
                                    'is_quarter_start',
                                    'is_quarter_end', 'is_year_start',
-                                   'is_year_end'],
```

Comment:
there are addl tests in ``tests/series/test_datetime_values.py`` (just add the new propery name)